client/signup/steps/jpo-site-type/style.scss

```diff
@@ -0,0 +1,295 @@
+.jpo-site-type {
+	position: relative;
+	text-align: center;
+}
+
+.jpo-site-type__image {
+	display: none;
+
+	@include breakpoint( ">480px" ) {
+		display: block;
+	}
+}
+
+@include breakpoint( ">480px" ) {
+	.jpo-site-type__choice-copy {
+		padding: 15px;
+		border-top: 1px solid transparentize( lighten( $gray, 20% ), .5 );
+	}
+}
+
+.jpo-site-type__cta {
+	color: $blue-wordpress;
+
+	@include breakpoint( "<480px" ) {
```

Comment:
Try adding `borderless` as a prop on mobile. It will do most of what is being done here.